symfony/src/Controller/DeceasedReportsController.php

```diff
@@ -0,0 +1,119 @@
+<?php
+namespace App\Controller;
+
+use App\Entity\DeceasedReport;
+use App\Form\DeceasedReportReviewType;
+use App\Form\DeceasedReportType;
+use App\Service\DeceasedReportsService;
+use App\Service\ParticipantSummaryService;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Session\SessionInterface;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route("/s/deceased-reports")
+ */
+class DeceasedReportsController extends AbstractController
+{
+    /**
+     * @Route("/", name="deceased_reports_index")
+     */
+    public function participantObservationIndex(Request $request, SessionInterface $session, DeceasedReportsService $deceasedReportsService) {
+        $statusFilter = $request->query->get('status', 'preliminary');
+        $organizationId = ($request->query->get('all', false) === false) ? $session->get('siteOrganizationId', null) : null;
```

Comment:
Should we allow for getting all reports? My thought here is that if the user doesn't have an organization associated with their current site for some reason, we should redirect them out of here.

Reply:
This was mostly for my debugging. Dropped.